packages/website/src/App.js

```diff
@@ -46,12 +46,12 @@ function App({ element, props }) {
       ) {
         ...mdxContent
       }
-      recipes: allMdx(
-        sort: { order: ASC, fields: [frontmatter___title] }
-        filter: { frontmatter: { recipe: { ne: null } } }
-      ) {
-        ...mdxContent
-      }
+      # recipes: allMdx(
```

Comment:
what did this use to do?

Reply:
Im gonna leave this out on purpose, but don't wanna delete the code. It used to show "Recipes" pages.